test/url-utils.test.js

```diff
@@ -11,6 +11,31 @@ test('geturl should return url without project if projectname is not provided',
   t.true(result === 'http://unleash-app.com/client/features');
 });
 
+test('geturl should return url with namePrefix if namePrefix is provided', t => {
+  const result = getUrl('http://unleash-app.com', '', 'unleash');
+  t.true(result === 'http://unleash-app.com/client/features?namePrefix=unleash');
+});
+
+test('geturl should return url with namePrefix and project both are provided', t => {
+  const result = getUrl('http://unleash-app.com', 'myProject', 'unleash');
+  t.true(result === 'http://unleash-app.com/client/features?namePrefix=unleash&namePrefix=unleash');
```

Comment:
Use the `t.is()` here will make it easier to spot the error. 

This line needs to change to:
`t.is(result, 'http://unleash-app.com/client/features?project=myProject&namePrefix=unleash');`